schedulearn-frontend/src/components/NestedList/NestedList.scss

```diff
@@ -45,6 +45,11 @@
   border: 1px solid #95a5a6;
 }
 
+.nested-list.disabled {
+  color: rgb(84, 84, 84);
+  background-color: rgb(235, 235, 228);
```

Comment:
Could you make this rgba(239, 239, 239, 0.3);
Then it would match with the colours of other disabled fields.